shell/packages/sandstorm-ui-topbar/topbar.js

```diff
@@ -427,6 +427,13 @@ SandstormTopbar.prototype.addItem = function (item) {
     throw new Error("duplicate top bar item name:", item.name);
   }
 
+  // We need inline style to conver the topbar "priority" into an "order" for the flexbox.
+  item.inlineStyle = {};
+  if (item.priority) {
+    item.inlineStyle['style'] = 'order: ' + item.priority + ';'
```

Comment:
And then this would just be `item.inlineStyle = 'order: ' + item.priority + ';'`